### ISSUE TITLE
Beta included by default while it should not be

### ISSUE THREAD

**Issue by rantoniuk** [raineorshine/npm-check-updates#1192]
`ncu --packageFile package.json --filter '@xxx/yyy' --target minor` results in:
 @xxx/yyy  ^5.0.0-beta-55  →  ^5.0.0-beta-9

However, by default beta packages should not be taken into consideration as the --pre should be 0 by default.
When --pre 0 is specified explicitly, the problem is solved.

```
ncu --version
16.1.3
node --version
v14.19.3
```

**Comment by raineorshine** [comment]
There is another case where `pre` is set to 1, which certainly should be better documented. That is if the existing version [is a prerelease itself](https://github.com/raineorshine/npm-check-updates/blob/768b5d353b5abce195c2c977a17b61280cdafd01/src/package-managers/filters.ts#L27). That is, if you are already using a prerelease, npm-check-updates assumes that you want to include prereleases in upgrades. As you discovered, you can specify `--pre 0` to opt out.

That said, I'm not understanding the *downgrade* from `^5.0.0-beta-55` to `^5.0.0-beta-9`. npm-check-updates should never downgrade versions by default. However, it's unclear to me why npm-check-updates would suggest `beta-9` when `beta-55` was available. Can you post the published versions of the relevant package?

```sh
npm view @xxx/yyy versions
```

**Comment by rantoniuk** [comment]
I think this is because -beta-9 was published after -beta-55  - yes, I know it's not the best practice ;-) I see now, so the [`inPre()`](https://github.com/raineorshine/npm-check-updates/blob/768b5d353b5abce195c2c977a17b61280cdafd01/src/lib/version-util.ts#L344) function follows the [prerelease spec format](https://semver.org/#spec-item-9) - that makes sense now. 

I guess then only issue is [the help message](https://github.com/raineorshine/npm-check-updates/blob/768b5d353b5abce195c2c977a17b61280cdafd01/src/cli-options.ts#L493) wrongly saying `--pre 0` by default.

**Comment by raineorshine** [comment]
Thanks! Updated help text published to `v16.2.1`.
